CoreScriptsRoot/CoreScripts/GamepadMenu.lua

```diff
@@ -111,6 +111,24 @@ local function setButtonEnabled(button, enabled)
 	radialButtons[button]["Disabled"] = not enabled
 end
 
+local function setButtonVisible(button, visible)
+	button.Visible = visible
+	if not visible then
+		setButtonEnabled(button, false)
+	end
+end
+
+local function enableVR()
+	local visibleButtons = {
+		Settings = true, LeaveGame = true,
+		PlayerList = false, Notifications = false,
+		Backpack = false, Chat = false
+	}
+	for button, _ in pairs(radialButtons) do
+		setButtonVisible(button, not not visibleButtons[button.Name])
```

Comment:
Just wanted to make sure it handles any potential new buttons not in the list. I highly doubt that'd happen though.